src/modules/river/layout.cpp

```diff
@@ -146,7 +146,9 @@ void Layout::handle_name(const char *name) {
     label_.hide();  // hide empty labels or labels with empty format
   } else {
     label_.show();
-    label_.set_markup(fmt::format(fmt::runtime(format_), Glib::Markup::escape_text(name).raw()));
+    std::string escaped_name=Glib::Markup::escape_text(name).raw();
```

Comment:
Looks like `escaped_name` isn't used. Can this line be deleted? Are there side effects?

Reply:
oh, don't remember really since it's almost been a year. But I think that was supposed to be used in the `getIcon()` function? I have been running this change for that same amount of time without issues. But then again I didn't change my layout names during that time.
It's probably fine to leave out?